setup.py

```diff
@@ -33,6 +33,15 @@
           "Operating System :: Microsoft :: Windows",
           "Operating System :: POSIX",
           "Programming Language :: Python",
+          "Programming Language :: Python :: 2",
+          "Programming Language :: Python :: 2.7",
+          "Programming Language :: Python :: 3",
+          "Programming Language :: Python :: 3.3",
```

Comment:
I juste removed 3.3 with 1ed25bda6a25f7f1bd4efdf9e6ed64c43afe592f

Could you remove it from here too?

Reply:
Removed!